jdk_8_maven/em/external/rest/ocvn/src/main/java/em/external/org/devgateway/ocvn/ExternalEvoMasterController.java

```diff
@@ -239,7 +239,12 @@ public SutInfoDto.OutputFormat getPreferredOutputFormat() {
 
     @Override
     public List<AuthenticationDto> getInfoForAuthentication() {
-        return Arrays.asList(AuthUtils.getForDefaultSpringFormLogin("ADMIN", "admin", "admin"));
+
+        // Adding a new user called user1 with password "password". User details are stored in the database.
```

Comment:
see previous comments. `Adding a new user called user1 with password "password"` can be removed. regarding database, can mention `init_db.sql` script in the comment here

Reply:
Hi Andrea, thanks for your comment. I fixed that.